sdk/keyvault/keyvault-admin/samples/v4/javascript/package.json

```diff
@@ -33,7 +33,7 @@
   "dependencies": {
     "@azure/keyvault-admin": "next",
     "dotenv": "latest",
-    "@azure/identity": "^1.1.0",
+    "@azure/identity": "2.0.0-beta.3",
```

Comment:
Ohh this is cool! Thank you!

Reply:
Actually do we want this to be using beta version in our samples? Seems like maybe we should use whatever is currently out there...

Reply:
This really shouldn’t be a problem! Up to you